frb_example/with_flutter/macos/Runner/bridge_generated.h

```diff
@@ -38,9 +38,7 @@ typedef struct wire_list_size {
 } wire_list_size;
 
 typedef struct WireSyncReturnStruct {
-  uint8_t *ptr;
-  int32_t len;
-  bool success;
+  uint8_t _opaque[0];
 } WireSyncReturnStruct;
```

Comment:
Because the dart bindings were manually coded in `io.dart`, I figured this header definition only serves indicating the existence of the type, whatever is inside! I will remove this anyway with the pointer change :smile: :bomb: